### ISSUE TITLE
Firewall devices not linking properly 

### ISSUE THREAD

**Issue by frenchtoasters** [linode/manager#9714]
When a Nodebalancer is attached to a Cloud Firewall, on the Cloud Firewall page the link to the Nodebalancer is incorrect. 

Current:
```
https://cloud.linode.com/linodes/447343/networking
```

Expected:
```
https://cloud.linode.com/nodebalancers/447343
```

**Comment by tyler-akamai** [comment]
Hey, thanks for the callout!

We have a separate [feature branch](https://github.com/linode/manager/tree/feature/firewall-nodebalancer) with all of the nodebalancer firewall updates (with the working links). The main develop branch does not currently have those changes.

**Comment by bnussman-akamai** [comment]
Resolved in Cloud Manager [v1.108.0](https://github.com/linode/manager/releases/tag/linode-manager%40v1.108.0) by   https://github.com/linode/manager/pull/9760 🎉
